TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace TestFramework.PageComponents
+{
+    public class CreateNewWaiterPageComponent 
+    {
+        IWebDriver driver { get; }
+
+        public CreateNewWaiterPageComponent(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        private IWebElement _nameField => driver.FindElement(By.Name("name"));
+        private IWebElement _emailField => driver.FindElement(By.Name("email"));
+        private IWebElement _passwordField => driver.FindElement(By.Name("password"));
+        private IWebElement _phoneNumberField => driver.FindElement(By.Name("phone_number"));
+        private IWebElement _addButton => driver.FindElement(By.XPath("//span[contains(text(),'Add')]/parent::button"));
+        private IWebElement _cancelButton => driver.FindElement(By.XPath("//span[contains(text(),'Cancel')]/parent::button"));
+
+
+        public CreateNewWaiterPageComponent ClickNameField()
+        {
+            _nameField.Click();
+            return this;
+        }
+        public CreateNewWaiterPageComponent ClickEmailField()
+        {
+            _emailField.Click();
+            return this;
+        }
+
+        public CreateNewWaiterPageComponent ClickPasswordField()
+        {
+            _passwordField.Click();
+            return this;
+        }
+        public CreateNewWaiterPageComponent ClickPhoneNumberField()
+        {
+            _phoneNumberField.Click();
+            return this;
+        }
+        public CreateNewWaiterPageComponent ClickAddToSubmitRegistrationWaiterButton()
+        {
+            _addButton.Click();
+            return new CreateNewWaiterPageComponent(driver); 
```

Comment:
I think you can't return the same page component because clicking on the Cancel button changes the page state to another page/component. Is it correct?